pkgpanda/build/__init__.py

```diff
@@ -48,6 +48,13 @@ def run(self, cmd):
         docker.append(self.container)
         docker += cmd
         check_call(docker)
+        DockerCmd.clean()
+
+    @staticmethod
+    def clean():
+        """Removes the last container that was spawned"""
+        container_id = check_output(["docker", "ps", "--latest", "-q"]).rstrip()
```

Comment:
This seems very likely to potentially accidentally pick up and kill the wrong docker container. The value of auto-cleaning here really to me isn't worth a chance of cleaning up some random unrelated docker.